src/frontend/pages/game.js

```diff
@@ -5,6 +5,10 @@ import React, { useState } from 'react';
 
 export default function Home() {
   const [takeVal, setTakeVal] = useState(10);
+  // state variable to track which color this user is once we hook this up to back end this will be dynamic
+  const [thisColor, setThisColor] = useState('Green');
```

Comment:
can we choose a variable name that more accurately represents what it is? I think when we're working with the data it's going to be confusing later on

Reply:
maybe: playerColor